app/views/work_orders/index.html.erb

```diff
@@ -0,0 +1,90 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Work Orders for 10/1/2019</h1>
+
+<div class="master">
+
+
+  <div>
+    <div><br></div>
+
+    <%  @minute.each do |hour| %>
+      <div class="flex_time" id="hour<%= hour %>"><%= hour %></div>
+
+
+    <%end %>
+  </div>
+
+
+  <% @technicians.each do |technician| %>
+    <section class="flex-items"  style="height: 1710px; position: relative" ><%= technician.name %>
+      <% @prior_work_time =  @work_orders[0].time%>
+      <% @prior_work_duration = 0 %>
+      <% @duration_height = 0 %>
+      <% @work_order_duration_height = 0 %>
+      <% @work_orders.each  do  |work_order|  %>
+        <% next unless work_order.technician_id == technician.id %>
```

Comment:
This should just be:

```<% technician.work_orders.each do |work_order| %>```

so that you don't have to iterate over the entire list for every technician.